guide/02-api-overview/release-notes.ipynb

```diff
@@ -1,5 +1,33 @@
 {
  "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# What's new in version 1.4.2\n",
+    "#### New\n",
+    "* Added ability to work with [StoryMap Journals](https://storymaps.arcgis.com/en/app-list/map-journal/) \n",
```

Comment:
Rephrase to "Added a new submodule [`storymap`](https://esri.github.io/arcgis-python-api/apidoc/html/arcgis.apps.storymap.html) under the `arcgis.apps` module to work with [ArcGIS Story Maps](https://storymaps.arcgis.com/en/app-list/map-journal/)